### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by dragibusnoir** [XilinJia/Podcini#96] [Ideas]
Hi, I have a few suggestions that I would like to see implemented in the app:

Be able to refresh podcasts from the playlist

Update podcasts at specific times

Have a "Select All" button in a podcast to mark all episodes as read

Use Weblate instead of Transifex

Thank you

**Comment by XilinJia** [comment]
> Have a "Select All" button in a podcast to mark all episodes as read

This feature currently exists

> Update podcasts at specific times

This can be planned.

> Be able to refresh podcasts from the playlist

This is specifically disabled. Explain your thought on why you need it.

> Use Weblate instead of Transifex

Transifex hasn't been useful. But explain you suggestion.

**Comment by dragibusnoir** [comment]
Yes, we can already schedule it, but I would like to do so at specific times. I know the release times of the podcasts I listen to, and it's pointless for the app to refresh every hour or on the hour... It's just to save battery, and no longer have to refresh manually.

My app directly opens the playlist to launch my episodes faster, but to refresh new episodes I have to go to the Subscriptions tab since I can't do it from the playlist.

I requested to help translate Podcini (into French) on Transifex several weeks ago, but my request is pending. Weblate is more accessible and user-friendly.

**Reply by XilinJia** [reply]
You can currently schedule automatic refresh at various intervals, though not at specific time, so that's not enough for you?

So you would like to do manual refresh of all subscriptions from within Queues view?

Sorry I didn't get notice on your request on Transifex (my bad for not checking often). If I accept your request there, would you still feel Transifex not easy to use?

Regarding translation, I know there are many new strings introduced and many obsolete in Podcini.  Perhaps you can suggest a way to get best updated strings?

**Reply by dragibusnoir** [reply]
Yes, I would like the podcasts to refresh, for example, at 4:10 PM, 5:20 PM, and 8:00 PM, and not have them refresh every hour or every 3 hours when it's unnecessary at certain times.


Yes, I would like to manually refresh the podcasts from the playlist.


To be honest, I've never used Transifex, I regularly use Weblate and sometimes Crowdin. On Weblate, it's indicated when a string has been modified and needs to be translated again, I don't know if that was your question.

**Comment by XilinJia** [comment]
refresh in Queues is added in 6.8.4, via the menu.  6.8.4 is a pre-release, if you don't want to risk, wait for the release later.
